pynamodb/models.py

```diff
@@ -335,112 +328,34 @@ def __repr__(self):
                 msg = "{0}<{1}>".format(self.Meta.table_name, serialized.get(HASH))
             return six.u(msg)
 
-    def delete(self, condition=None, conditional_operator=None, **expected_values):
+    def delete(self, condition=None):
         """
         Deletes this object from dynamodb
         """
-        self._conditional_operator_check(conditional_operator)
         args, kwargs = self._get_save_args(attributes=False, null_check=False)
-        if len(expected_values):
-            kwargs.update(expected=self._build_expected_values(expected_values, DELETE_FILTER_OPERATOR_MAP))
-        kwargs.update(conditional_operator=conditional_operator)
-        kwargs.update(condition=condition)
-        return self._get_connection().delete_item(*args, **kwargs)
 
-    def update_item(self, attribute, value=None, action=None, condition=None, conditional_operator=None, **expected_values):
-        """
-        Updates an item using the UpdateItem operation.
+        # TODO: raise exception if old args passed?
```

Comment:
python should have you covered:
`TypeError: delete() got an unexpected keyword argument 'conditional_operator'`